service/importer/definition.go

```diff
@@ -1,108 +1,111 @@
 package importer
 
+// ConfigurationDependencyKey is the reserved key of the service's configuration in the dependencies array.
+const ConfigurationDependencyKey = "service"
+
 // ServiceDefinition represents MESG services configurations.
 type ServiceDefinition struct {
 	// Name is the service name.
-	Name string `yaml:"name" json:"name,omitempty"`
+	Name string `yaml:"name" json:"name,omitempty" validate:"required,printascii,min=1"`
 
 	// Sid is the service id. It must be unique.
-	Sid string `yaml:"sid" json:"sid,omitempty"`
+	Sid string `yaml:"sid" json:"sid,omitempty" validate:"printascii,max=39"`
```

Comment:
```suggestion
	Sid string `yaml:"sid" json:"sid,omitempty" validate:"printascii,max=32"`
```

Reply:
Ok I have update on this:

- based on [rfc2181](https://tools.ietf.org/html/rfc2181#section-11) and [stackoverflow answer](https://stackoverflow.com/questions/2180465/can-domain-name-subdomains-have-an-underscore-in-it)
dns names can have under score.

- there is implementation in golang [net](https://github.com/golang/go/blob/master/src/net/dnsclient.go#L63) package, which we can use as is already working (we need only copy it to our src because it isn't exported)

Reply:
@krhubert can you show the new regex that needs to be use?

Reply:
There is no regexp, look at the net package and the `for` loop. Regexp is not needed